cvat/apps/consensus/views.py

```diff
@@ -0,0 +1,475 @@
+                ).distinct()
+            else:
+                perm = ConsensusReportPermission.create_scope_list(self.request)
+                queryset = perm.filter(queryset)
+
+            if target := self.request.query_params.get("target", None):
+                if target == ConsensusReportTarget.JOB:
+                    queryset = queryset.filter(job__isnull=False)
+                elif target == ConsensusReportTarget.TASK:
+                    queryset = queryset.filter(job__isnull=True)
+                else:
+                    raise ValidationError(
+                        "Unexpected 'target' filter value '{}'. Valid values are: {}".format(
+                            target, ", ".join(m[0] for m in ConsensusReportTarget.choices())
+                        )
+                    )
+
+        return queryset
+
+    CREATE_REPORT_RQ_ID_PARAMETER = "rq_id"
+
+    @extend_schema(
+        operation_id="consensus_create_report",
+        summary="Create a consensus report",
+        parameters=[
+            OpenApiParameter(
+                CREATE_REPORT_RQ_ID_PARAMETER,
+                type=str,
+                description=textwrap.dedent(
+                    """\
+                    The report creation request id. Can be specified to check the report
+                    creation status.
+                """
+                ),
+            )
+        ],
+        request=ConsensusReportCreateSerializer(required=False),
+        responses={
+            "201": ConsensusReportSerializer,
+            "202": OpenApiResponse(
+                RqIdSerializer,
+                description=textwrap.dedent(
+                    """\
+                    A consensus report request has been enqueued, the request id is returned.
+                    The request status can be checked at this endpoint by passing the {}
+                    as the query parameter. If the request id is specified, this response
+                    means the consensus report request is queued or is being processed.
+                """.format(
+                        CREATE_REPORT_RQ_ID_PARAMETER
+                    )
+                ),
+            ),
+            "400": OpenApiResponse(
+                description="Invalid or failed request, check the response data for details"
+            ),
+        },
+    )
+    def create(self, request, *args, **kwargs):
+        self.check_permissions(request)
+        input_serializer = ConsensusReportCreateSerializer(data=request.data)
+        input_serializer.is_valid(raise_exception=True)
+        task_id = input_serializer.validated_data["task_id"]
+
+        queue_name = settings.CVAT_QUEUES.CONSENSUS.value
+        queue = django_rq.get_queue(queue_name)
+        # rq_id = request.query_params.get(self.CREATE_REPORT_RQ_ID_PARAMETER, None)
+        rq_id = request.data.get("rq_id", f"merge_consensus:task.id{task_id}-by-{request.user}")
```

Comment:
Please move the parameter into query params.

Reply:
The `rq_id` will be the same for a particular task and user, so if the current query parameters are the same, the same `rq_id` will also be formed. So, should I still add it to query params?

Reply:
I'm not sure what the question is about. I'm just asking to move the parameter from the request body to the query, because in all other similar places it's passed in the query.